frontend/packages/topology/integration-tests/support/pages/topology/topology-page.ts

```diff
@@ -76,7 +76,9 @@ export const topologyPage = {
     });
   },
   verifyToplogyPageNotEmpty: () => {
-    cy.get('*[class="odc-topology"]').then(($el) => {
+    // Wait for topology page to load completely - increased timeout for yup async validation
+    cy.get('[class*="odc-topology"]', { timeout: 60000 }).should('be.visible');
+    cy.get('[class*="odc-topology"]').then(($el) => {
```

Comment:
Although not a new, could simplify the test.

```
    cy.get('.odc-topology', { timeout: 60000 })
      .should('be.visible')
      .within(() => {
        cy.get('[data-test="no-resources-found"]').should('not.exist');
      });
```
Additionally, could reduce the timeout if that works.